cmake/dependencies.cmake

```diff
@@ -83,37 +83,30 @@ endif()
 
 # libcudacxx -- get an explicit lubcudacxx build, matx tries to pull a tag that doesn't exist.
 # =========
-set(LIBCUDACXX_VERSION "1.8.0" CACHE STRING "Version of libcudacxx to use")
-include(deps/Configure_libcudacxx)
+morpheus_utils_configure_libcudacxx()
```

Comment:
Cant comment on the right line, but is this still needed? 
```
# Needed force rapids cpm to use our source directory.
set(CPM_SOURCE_CACHE "${CMAKE_SOURCE_DIR}/.cache/cpm")
# Prevent cpm_init from trying to tell us where to put cpm.cmake
include(get_cpm)
# Cant use rapids_cpm_init() for now since the `rapids_cpm_download()` creates a
# new scope when importing CPM. Manually do the other commands and import CPM on
# our own with get_cpm
#include("${rapids-cmake-dir}/cpm/detail/load_preset_versions.cmake")
#rapids_cpm_load_preset_versions()
```
Odd that morpheus requires it but MRC doesnt

Reply:
I think so; I remember trying to remove it and having problems. I think it might be libcudacxx that is setting `CPM_SOURCE_CACHE`. But I'll double check.